lib/kafka_ex/consumer_group/heartbeat.ex

```diff
@@ -79,6 +79,11 @@ defmodule KafkaEx.ConsumerGroup.Heartbeat do
       %HeartbeatResponse{error_code: error_code} ->
         Logger.warn("Heartbeat failed, got error code #{error_code}")
         {:stop, {:shutdown, {:error, error_code}}, state}
+
+      {:error, reason} ->
```

Comment:
It's not possible for this value to be returned from `KafkaEx.heartbeat/2` according to dialyzer.

Reply:
I'm sure it's possible, I've checked this on live application.

KafkaEx.heartbeat is called here - 
https://github.com/kafkaex/kafka_ex/blob/a746114bd2a673b438bb88234eb7966d556288df/lib/kafka_ex/consumer_group/heartbeat.ex#L69
->
KafkaEx.heartbeat  https://github.com/kafkaex/kafka_ex/blob/a746114bd2a673b438bb88234eb7966d556288df/lib/kafka_ex.ex#L146
->
KafkaEx.Server.handle_call({:heartbeat... - https://github.com/kafkaex/kafka_ex/blob/a746114bd2a673b438bb88234eb7966d556288df/lib/kafka_ex/server.ex#L336
->
KafkaEx.Server0P9P0.kafka_server_heartbeat  - https://github.com/kafkaex/kafka_ex/blob/a746114bd2a673b438bb88234eb7966d556288df/lib/kafka_ex/server_0_p_9_p_0.ex#L186
->
KafkaEx.Server0P9P0.consumer_group_sync_request - https://github.com/kafkaex/kafka_ex/blob/a746114bd2a673b438bb88234eb7966d556288df/lib/kafka_ex/server_0_p_9_p_0.ex#L228

Reply:
heh, that's unfortunate. I suspect we need to fix the `@spec` for KafkaEx.heartbeat then, so that dialyzer will pass.